docs/faq.rst

```diff
@@ -129,7 +129,12 @@ Does Scrapy crawl in breadth-first or depth-first order?
 
 By default, Scrapy uses a `LIFO`_ queue for storing pending requests, which
 basically means that it crawls in `DFO order`_. This order is more convenient
-in most cases. If you do want to crawl in true `BFO order`_, you can do it by
+in most cases.
+
+However, when the number of start request is small it always `crawls in BFO
```

Comment:
I don't think it is correct: if there is a single request in start_requests, but then spider schedules many requests (e.g. after extracting links), behavior would be as usual, maybe besides first few requests.